README.md

```diff
@@ -111,6 +111,14 @@ const {vp} = await vqpr.fromQrCode({text: qrCodeText, documentLoader});
 // becomes a JSON object with the VP in the example above.
 ```
 
+### Encoding Options
+
+- `qrMultibaseEncoding`: Encoding for QR data. `B` for [RFC4648][] base32, `R`
+  for [RFC9285][] base45. base45 is more efficient. Defaults to base32.
+- `qrErrorCorrectionLevel`: Error correction level used in the QR code. 'L' Low
+  7%, 'M' Medium 15%, 'Q' Qartile 25%, 'H' High 30%. Defaults to 'L'.
+- `qrVersion`: QR version. 1-40 or 0 for auto. Defaults to auto.
```

Comment:
Another way to do this would be under the `options.qr` namespace (e.g., `options.qr.multibaseEncoding`, ...). I'm ok with either.

Reply:
Suffering analysis paralysis with this and differences in libraries and use cases.  Going to go with something for now and will adjust in major releases later as needed and experience is gained.